client/src/pages/CustomerProfile.js

```diff
@@ -1,14 +1,164 @@
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import { AuthContext } from "../context/auth-context";
+import { Paper, Grid, Button, makeStyles } from "@material-ui/core";
+
+import Map from "../components/Map";
+
+import profilePic from "../assets/dummyavatar.png";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Can you add a bottom margin to the card so it's not flush with the bottom?

Reply:
Sure! It was flush with the bottom in the design mockups, so I followed that.